### DISCUSSION TITLE
Relay Components show infinitely increasing load while simultaneously not allowing any power to pass through.

### DISCUSSION THREAD

**Discussion by SeaverBeaver** [FakeFishGames/Barotrauma#15030] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

After the Unto the Breach update, I noticed that the machines on board my submarine were no longer receiving power through their respective relays, and would only be powered once the relays had been cycled off and on. When inspecting the relays, I found the following:

https://github.com/user-attachments/assets/87a20c86-d6c1-4290-8d91-b17e4eba0e50

![image](https://github.com/user-attachments/assets/95681ec0-8858-4cb1-aedd-62ef7861e9f2)
![image](https://github.com/user-attachments/assets/9c9de12d-c1f6-432d-96ae-573ace5d4503)

An online relay connected to any machines attempting to draw power through it would show infinitely increasing load while simultaneously showing no power input through the electricians goggles, and would allow no power to pass through.

**This issue can be fixed by resetting the relay to prefab.**

I dug through the UTB patch notes and found no mention of any changes to relays. Maybe something undocumented?
In any case, while the problem can be retroactively fixed by resetting the relay to prefab, this means that any subs built before Unto the Breach could potentially have problems.

### Reproduction steps

1. The unique nature of this bug means to reproduce it you'll have to load a sub built before Unto the Breach, with relays supplying power to machines. So far as I can tell no vanilla subs have this configuration, so they seem to be in the clear.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Ek-Videogames** [comment]
I actually was noticing this issue as well - with an even much simpler relay system.
( only one relay, used as a toggleable output from a shuttle's battery. )

In my case the issue was the relay wouldn't transmit power after the shuttle undocked from the ship, until it was switched off and on again.

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.
